autoflake.py

```diff
@@ -986,6 +986,125 @@ def find_files(filenames, recursive, exclude):
                 _LOGGER.debug('Skipped %s: matched to exclude pattern', name)
 
 
+def process_pyproject_toml(toml_file_path):
+    """Extract config mapping from pyproject.toml file."""
+    import toml
+
+    return toml.load(toml_file_path).get('tool', {}).get('autoflake', None)
+
+
+def process_setup_cfg(cfg_file_path):
+    """Extract config mapping from setup.cfg file."""
+    import configparser
+
+    reader = configparser.ConfigParser()
+    reader.read(cfg_file_path)
+    if not reader.has_section("autoflake"):
+        return None
+
+    return reader["autoflake"]
+
+
+if sys.version_info > (3, 5, 0):
+    _commonpath = os.path.commonpath
+else:
+    def _commonpath(paths):
+        if not paths:
+            raise ValueError("commonpath() arg is an empty sequence")
+        is_abs = os.path.isabs(paths[0])
+        if not all(is_abs is os.path.isabs(p) for p in paths[1:]):
+            raise ValueError("paths are mixed absolute and relative")
+        drive = os.path.splitdrive(paths[0])[0]
+        if not all(drive == os.path.splitdrive(p)[0] for p in paths[1:]):
+            raise ValueError("paths are on different drives")
+        common = os.path.commonprefix(paths)
+        len_ = len(common)
+        s = {os.path.sep, os.path.altsep}
+        if len_ and any(len(p) > len_ and p[len_] not in s for p in paths):
+            common = os.path.split(common)[0]
+        return common
```

Comment:
Ops, missed this one, we can also simplify it. Minimum requirement is Python 3.7 now.

Reply:
ok, removed that